### PR TITLE
Add, check, and delete to-dos

### PR DESCRIPTION
**Pull request by abbas-13**
Screenshot 2024-01-20 at 15 30 03

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,77 @@
...
+
+      return updatedToDos;
+    });
+  };
+
+  useEffect(() => {
+    setToDos(JSON.parse(localStorage.getItem("toDoData")) || []);
+  }, []);
+
+  console.log({ toDos });
```
Comment (mohammad-23) [inline src/App.js:42]: remove this console log

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
       <Appshell>
-        <button className="m-2 shadow-md hover:drop-shadow-xl hover:bg-blue-700 p-2 w-24 rounded-lg bg-blue-500 text-white">
-          Add +
-        </button>
-        <ToDo />
+        <h2 className="text-2xl font-semibold p-2">Reminders</h2>
+        <div className="border border-gray-200 m-2"></div>
+        <div className="grid grid-cols-8">
+          <button
+            onClick={handleClick}
```
Comment (mohammad-23) [inline src/App.js:51]: rename this to a more readable name like on AddTodo

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,77 @@
...
+            Add +
+          </button>
+        </div>
+        <div className="grid grid-rows-5">
+          {toDos &&
+            toDos.map((toDo, index) => (
+              <ToDoItem
+                key={index}
+                data={toDo}
+                handleChange={handleChange}
```
Comment (mohammad-23) [inline src/App.js:63]: this should have a more readable name like onTodoCheck

File: src/Components/To-DoItem.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { MdDeleteSweep } from "react-icons/md";
+
+export const ToDoItem = ({ data, handleChange, deleteToDo }) => {
+  const toDoClassName = data.isChecked
+    ? "font-semibold text-gray-600 ml-2 line-through"
+    : "ml-2 font-semibold";
+  console.log(data);
```
Comment (mohammad-23) [inline src/Components/To-DoItem.js:7]: remove this console log

File: src/App.js (hunk 1 above)
Comment (abbas-13) [reply src/App.js:42]: removed

File: src/Components/To-DoItem.js (hunk 4 above)
Comment (abbas-13) [reply src/Components/To-DoItem.js:7]: removed

File: src/App.js [hunk 5]
```diff
@@ -1,14 +1,69 @@
...
+        <div className="grid grid-cols-8">
+          <button
+            onClick={toggleModal}
+            className="m-2 shadow-md hover:drop-shadow-xl active:bg-blue-700 p-2 rounded-xl bg-blue-500 text-white"
+          >
+            Add +
+          </button>
+        </div>
+        <div className="grid grid-rows-5">
+          {toDos &&
```
Comment (mohammad-23) [inline src/App.js:50]: if you initialize the `todos` state with [], then you don't need to add `todos &&`

**Review by mohammad-23 — APPROVED** [review]

**Pull request merged**